src/docs/asciidoc/getting_started.adoc

```diff
@@ -1,684 +1,342 @@
+[source,python,indent=0,subs="verbatim,attributes",role="secondary"]
+.Python
 ----
-java --add-modules java.se \
-  --add-exports java.base/jdk.internal.ref=com.hazelcast.core \
-  --add-opens java.base/java.lang=com.hazelcast.core \
-  --add-opens java.base/java.nio=com.hazelcast.core \
-  --add-opens java.base/sun.nio.ch=com.hazelcast.core \
-  --add-opens java.management/sun.management=com.hazelcast.core \
-  --add-opens jdk.management/com.ibm.lang.management.internal=com.hazelcast.core \
-  --add-opens jdk.management/com.sun.management.internal=com.hazelcast.core \
-  --module-path lib \
-  --module com.hazelcast.core/com.hazelcast.core.server.HazelcastMemberStarter
+# Installation notes: Run the following command
+# pip install hazelcast-python-client
+
+import hazelcast
+
+if __name__ == "__main__":
+    # Start the client and connect to the cluster
+    hz = hazelcast.HazelcastClient()
+    # Create a Distributed Map in the cluster
+    map = hz.get_map("my-distributed-map").blocking()
+    # Standard Put and Get
+    map.put("1", "John")
+    map.put("2", "Mary")
+    map.put("3", "Jane")
+    # Shutdown the client
+    hz.shutdown()
 ----
 
-_This example expects `hazelcast-{imdg-version}.jar` placed in the `lib` directory._
+[source,go,indent=0,subs="verbatim,attributes",role="secondary"]
+.Go
+----
+//Installation notes: Run the following command
+//go get github.com/hazelcast/hazelcast-go-client
+
+import "github.com/hazelcast/hazelcast-go-client"
+
+func mapSampleRun() {
+	// Start the client and connect to the cluster
+	hz, _ := hazelcast.NewClient()
+	// Create a Distributed Map in the cluster
+	mp, _ := hz.GetMap("myDistributedMap")
+	//Standard Put and Get
+	mp.Put("1", "John")
+	mp.Put("2", "Mary")
+	mp.Put("3", "Jane")
+	// Shutdown the client
+	hz.Shutdown()
+} 
+----
 
-[[starting-the-member-and-client]]
-=== Starting the Member and Client
+For comprehensive information on the clients, see the following sections:
 
-Having installed Hazelcast, you can get started.
+* <<java-client, Java client>>
+* link:https://github.com/hazelcast/hazelcast-cpp-client[C++ client]
+* link:https://github.com/hazelcast/hazelcast-csharp-client[C# client]
+* link:https://github.com/hazelcast/hazelcast-nodejs-client[Node.js client]
+* link:https://github.com/hazelcast/hazelcast-python-client[Python client]
+* link:https://github.com/hazelcast/hazelcast-go-client[Go client]
 
-In this short tutorial, you perform the following activities:
+=== Connecting Management Center to the Cluster
 
-. Create a simple Java application using the Hazelcast distributed map and queue.
-. Run our application twice to have a cluster with two members (JVMs).
-. Connect to our cluster from another Java application by using the
-Hazelcast Native Java Client API.
+Hazelcast Management Center helps you to monitor and manage your IMDG cluster.
+After you created your cluster and client application as depicted in the above sections,
+let's connect Management Center.
```

Comment:
let's connect Management Center to the cluster? Maybe also mention somehow here that the clients are irrelevant for the Management center and that you can connect it to the cluster without any other clients?